ci-operator/config/openshift/windows-machine-config-operator/openshift-windows-machine-config-operator-community-4.10__okd.yaml

```diff
@@ -59,7 +59,23 @@ tests:
 - as: aws-e2e-operator
   steps:
     cluster_profile: aws
+    env:
+      COMMUNITY: "true"
     test:
+    - as: test
```

Comment:
Could we name this step something more explanatory like `pre` or `setup-community`?

Reply:
changed to setup-community